glide.yaml

```diff
@@ -2,6 +2,9 @@ package: github.com/kubeless/kubeless
 import:
 - package: github.com/sirupsen/logrus
 - package: github.com/spf13/cobra
+  version: b95ab734e27d33e0d8fbabf71ca990568d4e2020
+- package: github.com/spf13/pflag
```

Comment:
it seems that you are not using `pflag`?

Reply:
That is part of #567 issue. The `cobra` package will pull down the `pflag` together and we must pin them to specific revisions. Otherwise it will download latest every `glide up` and the kubeless cli won't built successfully

Reply:
mm, I am not sure I am following, shouldn't `cobra` specify its desired `pflag` version?

Reply:
I guess `cobra` did. But for reasons when I removed them in the vendor folder and just `glide up --skip-test -v` again, I got the a different revision of `pflag` (compared to the one we have) and I couldn't build the cli. So to be safe, I pinned both packages to revisions we are having.